oneview_redfish_toolkit/api/computer_system_collection.py

```diff
@@ -59,10 +61,10 @@ def _set_redfish_members(self):
             Populates self.redfish["Members"] with the links to Redfish
             ComputerSystems.
         """
-        for server_hardware, index in \
-                zip(self.server_hardware,
-                    range(len(self.server_hardware))):
-
+        for server_profile, index in \
```

Comment:
just a note: Could we do this iteration using a simple loop `for` without apply a `zip` method to the `self.server_profiles` and without `index` variable? This will keep the code more simple, no? What do you think?

For example:
```python
for item in self.server_profiles:
    dict_result = {"@odata.id": item["serverProfileUri"].split("/")[-1]}
    self.redfish["Members"].append(dict_result)
```